addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -313,3 +401,35 @@ func (r *MirrorPeerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&multiclusterv1alpha1.MirrorPeer{}, builder.WithPredicates(mpPredicate)).
 		Complete(r)
 }
+
+func (r *MirrorPeerReconciler) labelRBDStorageClasses(ctx context.Context, mp multiclusterv1alpha1.MirrorPeer, storageClusterNamespace string) []error {
+	// Get all StorageClasses in storageClusterNamespace
+	scs := &storagev1.StorageClassList{}
+	klog.Infof("Getting all StorageClasses in namespace %q", storageClusterNamespace)
+	err := r.SpokeClient.List(ctx, scs, client.InNamespace(storageClusterNamespace))
+	var errs []error
+	if err != nil {
+		errs = append(errs, err)
+		return errs
+	}
+	klog.Infof("Found %d StorageClasses in namespace %q", len(scs.Items), storageClusterNamespace)
+	key := r.SpokeClusterName
+	for _, sc := range scs.Items {
+		if _, ok := clusterFSIDs[key]; !ok {
```

Comment:
instead of reading from the map, if you store it in a variable, it will be more efficient.

Reply:
I feel better with a map, i could check for the existence of the key.
I doubt there is any performance implications.